src/main/python/docs/source/federated.rst

```diff
@@ -0,0 +1,96 @@
+.. ------------------------------------------------------------------------------
+..  Copyright 2020 Graz University of Technology
+..
+..  Licensed under the Apache License, Version 2.0 (the "License");
+..  you may not use this file except in compliance with the License.
+..  You may obtain a copy of the License at
+..
+..    http://www.apache.org/licenses/LICENSE-2.0
+..
+..  Unless required by applicable law or agreed to in writing, software
+..  distributed under the License is distributed on an "AS IS" BASIS,
+..  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+..  See the License for the specific language governing permissions and
+..  limitations under the License.
+.. ------------------------------------------------------------------------------
+
+
+Federated SystemDS
+==================
+
+The python SystemDS supports federated execution.
+To enable this, each of the federated environments have to have 
+a running federated worker.
+
+Start Federated worker
+----------------------
+
+To start a federated worker:
+
+  ./bin/systemds.sh WORKER 8001
+
+Simple Aggregation Example
+--------------------------
+
+In this example we use a single Federated worker, and aggregate the sum of its data.
+
+First we need to create some data for our federated worker to use.
+In this example we simply use Numpy to create a ``test.csv`` file.::
+
+  # Import numpy
+  import numpy as np
+  a = np.asarray([[1,2,3],[4,5,6],[7,8,9]])
+  np.savetxt("temp/test.csv", a, delimiter=",")
+
+Currently we also require a metadata file for the federated worker.
+This should be located next to the ``test.csv`` file, containing::
+
+  {
+    "data_type": "matrix",
+	"format": "csv",
+	"header": false,
+	"rows":3,
+	"cols":3 
+  }
+
+After creating our data we start a Federated worker instance the 
+federated instructions can be executed. 
```

Comment:
Does Federated need the capital F? Leaving out one of the federated would also be a good option. And there's an "on" missing at the end of the sentence (...start an instance... the instructions can be executed *on*). Maybe it's not *on*, but you execute on/in/at an instance. Or you make two sentences where you start in the first and execute in the second one.